fastf1/core.py

```diff
@@ -2189,46 +2238,49 @@ def pick_teams(self, names):
         """
         return self[self['Team'].isin(names)]
 
-    def pick_fastest(self, only_by_time=False):
+    def pick_fastest(self, round="all"):
```

Comment:
We can't just remove `only_by_time` because of backwards-compatibility. That would need a proper deprecation warning first and followed by a removal in a later release. 
Apart from that, it still has a use to differentiate between deleted and not deleted laps. Therefore, it should stay and work exactly as before.